recipes/phylip/LICENSE

```diff
@@ -0,0 +1,34 @@
+# <From the original phylip website>
+#
+# PHYLIP, the Phylogeny Inference Package, is a package of programs
```

Comment:
This needs to be in the unpacked tarball to be recognised for `license_file` afaik.